src/clojure/contrib/inflect.cljc

```diff
@@ -11,8 +12,8 @@
 
 (defn pluralize-noun [count noun]
   "Return the pluralized noun if the `count' is
```

Comment:
Lets add a `pre` contract to `pluralize-noun` that `count` cannot be less than zero.